src/aave-v2/SupplyVault.sol

```diff
@@ -10,12 +10,21 @@ import {SupplyVaultBase} from "./SupplyVaultBase.sol";
 /// @custom:contact security@morpho.xyz
 /// @notice ERC4626-upgradeable Tokenized Vault implementation for Morpho-Aave V2.
 contract SupplyVault is ISupplyVault, SupplyVaultBase {
-    /// UPGRADE ///
+    /// STORAGE ///
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    uint256[50] private __gap;
+
+    /// CONSTRUCTOR ///
 
     /// @dev Initializes network-wide immutables.
     /// @param _morpho The address of the main Morpho contract.
     constructor(address _morpho) SupplyVaultBase(_morpho) {}
```

Comment:
This is not needed

Reply:
why? We must initiate the immutable variable

Reply:
The contract already inherits from `SupplyVaultBase` and by defaults inherits from the constructor (we're not changing it here) :p

Reply:
mh ok my bad

Reply:
Made me realize that storage variables could be immutables on Morpho-Compound